python_orocos_kdl/manifest.xml

```diff
@@ -1,22 +0,0 @@
-<package>
-  <description brief="python_orocos_kdl">
-
-     python_orocos_kdl
-
-  </description>
-  <author>Ruben Smits</author>
-  <license>LGPL</license>
-  <review status="unreviewed" notes=""/>
-  <url>http://ros.org/wiki/python_orocos_kdl</url>
-
-  <depend package="orocos_kdl"/>
-  <rosdep name="python-sip"/>
-
-  <export>
-    <python path="${prefix}/lib"  />
-    <rosdoc config="rosdoc.yaml"/>
-  </export>
-
-</package>
-
-
```

Comment:
Be aware that `manifest.xml` is not only used by older ROS versions, but also by other build tools used by the Orocos community, like [Autoproj](https://github.com/rock-core/autoproj). We also parse it in [rtt/UseOROCOS-RTT-helpers.cmake](https://github.com/orocos-toolchain/rtt/blob/49752330ac2290ffb5206d42245ace716e2d93db/UseOROCOS-RTT-helpers.cmake#L54-L93). The two specs used to be compatible a long time ago, but also Autoproj added new features that were not understood by the ROS tools.

The dual use of `manifest.xml` already caused some trouble in the past. I once proposed to prefer `package.xml` if it exists (https://github.com/ros-infrastructure/rosdep/issues/353), but that was rejected back then. I noticed that the issue surfaced again recently (https://github.com/ros-infrastructure/rosdep/pull/737).

In this particular case it should be fine to remove the `manifest.xml` as `python_orocos_kdl` is not used by Rock if I am not mistaken (@doudou?).

Reply:
> In this particular case it should be fine to remove the manifest.xml as python_orocos_kdl is not used by Rock if I am not mistaken (@doudou?).

I don't for sure (only use KDL C++). Not sure about the rest of the Rock users. I'll drop a line on the mailing list.